src/command_modules/vsts-cli-release/vsts/cli/vrelease/_format.py

```diff
@@ -0,0 +1,54 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import dateutil
+
+
+from collections import OrderedDict
+
+
+def transform_releases_table_output(result):
+    table_output = []
+    for item in result:
+        table_output.append(_transform_release_row(item))
+    return table_output
+
+
+def transform_release_table_output(result):
+    table_output = [_transform_release_row(result)]
+    return table_output
+
+
+def _transform_release_row(row):
+    table_row = OrderedDict()
+    table_row['ID'] = row['id']
+    table_row['Name'] = row['name']
+    table_row['Status'] = row['status']
+    table_row['Definition ID'] = row['releaseDefinition']['id']
+    table_row['Definition Name'] = row['releaseDefinition']['name']
+    table_row['Created on'] = row['createdOn']
+    table_row['Reason'] = row['reason']
+    return table_row
+
```

Comment:
Table outputs are supposed to be confined such that they do not overflow to a new line on an average resolution screen. 
Bottom line being keep it as short as possible such that it is useful for human user. 

In the above fields I think we can get rid of IDs if we are outputting names. If there are commands which will not accept names but only Ids then you will need to display Ids so user can copy and use them in other commands.

Can you share a sample output for this?

Reply:
![sample-output](https://user-images.githubusercontent.com/13657698/48397928-533bfc80-e745-11e8-9c47-aaa62520c517.PNG)